api/lecture_halls.go

```diff
@@ -209,13 +210,28 @@ func (r lectureHallRoutes) lectureHallIcal(c *gin.Context) {
 		})
 		return
 	}
+	err = c.Request.ParseForm()
+	if err != nil {
+		_ = c.Error(tools.RequestError{Status: http.StatusBadRequest, CustomMessage: "Bad Request", Err: err})
+		return
+	}
+	lectureHallsStr := strings.Split(c.Request.Form.Get("lecturehalls"), ",")
```

Comment:
One last thing I noticed: This does not work if all lecture halls are deselected, resulting in the following error:
<img width="547" alt="grafik" src="https://user-images.githubusercontent.com/30267166/197364655-eb2415fe-46da-430b-9cdc-808830c5d83a.png">